.circleci/config.yml

```diff
@@ -2,20 +2,43 @@ version: 2.1
 orbs:
   slack: circleci/slack@3.4.2
 
-jobs:
-  build:
+executors:
+  tap_tester_mongo_4_2:
     docker:
       - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
-      - image: mongo:4.2.0-bionic
+      - image: singerio/mongo:4.2-bionic
         environment:
           MONGO_INITDB_ROOT_USERNAME: dev
           MONGO_INITDB_ROOT_PASSWORD: Password1
-        command: [mongod, --replSet, rs0]
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+  tap_tester_mongo_4_4:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
+      - image: singerio/mongo:4.4-bionic
+        environment:
+          MONGO_INITDB_ROOT_USERNAME: dev
+          MONGO_INITDB_ROOT_PASSWORD: Password1
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+  tap_tester_mongo_5_0:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
+      - image: singerio/mongo:5.0
+        environment:
+          MONGO_INITDB_ROOT_USERNAME: dev
+          MONGO_INITDB_ROOT_PASSWORD: Password1
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+
+jobs:
+  build:
+    executor: tap_tester_mongo_4_2
     steps:
       - checkout
       - run:
           name: 'Install Dockerize'
-          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          command: |
+            wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
```

Comment:
We can pretty safely remove the `\` and `&&` in this section. The `&&` gave us the ability to do multiple commands in the `command line`. But now that we are in a `command block`, we can just have multiple commands

Reply:
Removed